### PR TITLE
Clean up several warn

### PR DESCRIPTION
**Pull request by paranlee**
This patch will redeem several warn #4028.

1. Comment format
2. Deprecated PointerType::getElementType()
3. use setuptools

Signed-off-by: Paran Lee <p4ranlee@gmail.com>

### CODE REVIEW

File: src/cc/bpf_module_rw_engine.cc [hunk 1]
```diff
@@ -416,7 +416,7 @@ int BPFModule::annotate() {
       StructType *st = dyn_cast<StructType>(t);
 #else
     if (PointerType *pt = dyn_cast<PointerType>(gvar->getType())) {
-      StructType *st = dyn_cast<StructType>(pt->getElementType());
+      StructType *st = dyn_cast<StructType>(pt->getPointerElementType());
```
Comment (chenhengqi) [inline src/cc/bpf_module_rw_engine.cc:419]: I think this will be addressed by #4020 .

Comment (paranlee) [reply src/cc/bpf_module_rw_engine.cc:419]: Thanks for guide! @chenhengqi :)

Comment (yonghong-song) [reply src/cc/bpf_module_rw_engine.cc:419]: Yes, just merged #4020, so the above change is not needed.

File: src/cc/frontends/clang/kbuild_helper.cc [hunk 2]
```diff
@@ -74,28 +74,29 @@ int KBuildHelper::get_flags(const char *uname_machine, vector<string> *cflags) {
...
+   *                 -I$(srctree)/arch/$(SRCARCH)/include \
+   *                 -I$(objtree)/arch/$(SRCARCH)/include/generated \
+   *                 $(if $(building_out_of_srctree),-I$(srctree)/include) \
+   *                 -I$(objtree)/include \
+   *                 $(USERINCLUDE)
+   *
+   * Some distros such as openSUSE/SUSE and Debian splits the headers between
+   * source/ and build/. In this case, just $(srctree) is source/ and
+   * $(objtree) is build/.
+   */
```
Comment (yonghong-song) [inline src/cc/frontends/clang/kbuild_helper.cc:99]: I think existing comment is okay, there is no need to change. This is C++ any way.

File: src/python/setup.py.in [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Copyright (c) PLUMgrid, Inc.
 # Licensed under the Apache License, Version 2.0 (the "License")
-from distutils.core import setup
+from setuptools import setup
```
Comment (yonghong-song) [inline src/python/setup.py.in:3]: could you explain why this change in the commit message?
